scipy/spatial/transform/_rigidtransformation.pyx

```diff
@@ -6,9 +6,59 @@ from ._rotation import Rotation
 
 cimport numpy as np
 cimport cython
+from cython.view cimport array
 
 np.import_array()
 
+# utilities for empty array initialization
```

Comment:
I had to copy this from _rotation.pyx. Can we expose the function _compose_quat for other internal use in scipy?